### ISSUE TITLE
Stray xpos tag: "Sw"

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_Italian#2]
sent_id = 2Parole_4-180
word 19

The tag here is Sw instead of SW:

```
17      film    film    NOUN    S       Gender=Masc     14      nmod    14:nmod:di      _
18      Les     Les     PROPN   SW      Foreign=Yes     17      nmod    17:nmod _
19      invasions       invasions       PROPN   Sw      _       18      flat:name       18:flat:name    _
20      barbares        barbares        PROPN   SW      Foreign=Yes     18      flat:name       18:flat:name    SpaceAfter=No
21      .       .       PUNCT   FS      _       5       punct   5:punct _
```

**Comment by AngledLuffa** [comment]
I have mailed this to the email address listed in "contributing" as well.

**Comment by AngledLuffa** [comment]
This has been fixed since 52a4a5c228ddb2576850a2f3d3029397bb368d07
